src/k8s-engine/lima.ts

```diff
@@ -866,6 +889,19 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
           return;
         }
 
+        const username = process.env.USER || process.env.LOGNAME || process.env.USERNAME;
```

Comment:
This sets the host username. You need the username inside the guest. That may be different when the macOS username is not a valid Linux username. In that case the guest username will be `lima`.

You should be able to get the name in the guest via `${USER}` or `$(id -nu)`.

Reply:
This isn't needed if we make the socket world-accessible